README.md

```diff
@@ -1,85 +1,210 @@
----------  ----                                      ------   -------     --------  ------------          --------  -----------      -------------
-openebs    pvc-cb978ab8-9045-4d40-abc5-98dfd4fd82fb  Healthy  master-dev  5Gi       cstor.csi.openebs.io  Attached  ReadWriteOnce    vanisingh
-openebs    pvc-e20c1212-1ef6-42c4-9638-0145fa3fb4f9  Healthy  master-dev  5Gi       N/A                   N/A                        N/A
-
-
-# Describe a single volume
-$ kubectl openebs describe volume pvc-cbe030cb-63ca-4dfd-ba57-7719a8c93fb2
-Volume Details :
-----------------
-Name            : pvc-cbe030cb-63ca-4dfd-ba57-7719a8c93fb2
-Access Mode     : ReadWriteOnce
-CSI Driver      : cstor.csi.openebs.io
-Storage Class   : openebs-csi-cstor-sparse
-Volume Phase    : Bound
-Version         : master-dev
-CSPC            : cspc-stripe
-Size            : 5Gi
-Status          : Healthy
-ReplicaCount	: 1
-
-Portal Details :
-----------------
-IQN             :  iqn.2016-09.com.openebs.cstor:pvc-cbe030cb-63ca-4dfd-ba57-7719a8c93fb2
-Volume          :  pvc-cbe030cb-63ca-4dfd-ba57-7719a8c93fb2
-TargetNodeName  :  vanisingh
-Portal          :  10.103.173.0:3260
-TargetIP        :  10.103.173.0
-
-Replica Details :
-----------------
-Name                                                        Pool Instance     Status
-----                                                        -------------     ------
-pvc-cbe030cb-63ca-4dfd-ba57-7719a8c93fb2-cspc-stripe-56pv   cspc-stripe-56pv  Healthy
-
-# Get CStor pools
-$ kubectl openebs get pools
-Name              Namespace  HostName                       Free    Capacity   ReadOnly  ProvisionedReplicas  HealthyReplicas  Status  Age
-----              ---------  --------                       ----    --------   --------  -------------------  ---------------  ------  ---
-fastssd-cstor     test       director-dev-cluster-1-node-1  48200M  48202370k  false     1                    1                ONLINE  2d5h
-```
-
+## Flags
+* `--openebs-namespace` :- to override the determination of `namespace` where storage engine is installed with the provided value.
+* `--namespace, -n` :- to pass the namespace, if the resource is namespaced, like `pvc` etc.
 
+## Usage
+* #### Get `cStor` volumes
+  ```bash
+  $ kubectl openebs get volumes
+  NAMESPACE   NAME                                       STATUS    VERSION    CAPACITY   STORAGE CLASS         ATTACHED   ACCESS MODE      ATTACHED NODE
+  cstor       pvc-193844d7-3bef-45a3-8b7d-ed3991391b45   Healthy   2.9.0      5.0 GiB    cstor-csi-sc          Bound      ReadWriteOnce    N/A
+  cstor       pvc-b84f60ae-3f26-4110-a85d-bce7ec00dacc   Healthy   2.0.0      20 GiB     common-storageclass   Bound      ReadWriteOnce    node1-virtual-machine
+  ```
+  Note: For volumes not attached to any application, the `ATTACH NODE` would be shown as `N/A`.
+
+* #### Get `cStor` pools
+  ```bash
+  $ kubectl openebs get volumes
```

Comment:
```suggestion
  $ kubectl openebs get pool
```

Reply:
Done.

Reply:
@vharsh Please add the new flag and its usage in the `Flags` and `Usage` after #28 is merged.